specs/kernel_info.table

```diff
@@ -1,5 +1,5 @@
 table_name("kernel_info")
-description("Basic active kernel information.")
+description("Basic active kernel information. Not available on ARM architectures.")
```

Comment:
Also, I would talk about Apple Silicon, not ARM, because on Linux this works fine.

Reply:
I don't think we should say anything. Many table behave slightly different across OS versions, platforms, and architectures. I don't think parenthetical comments is a viable way to express that -- we'd need to apply it everywhere and it would become noise.